Composer/packages/server/src/models/settings/defaultSettingManager.ts

```diff
@@ -103,6 +104,10 @@ export class DefaultSettingManager extends FileSettingManager {
         isSkill: true,
         allowedCallers: [],
       },
+      crossTrain: {
```

Comment:
Terminology: 
Inter --> across
Intra --> Inner/within/internal
So we should have:
Inter / Intra  or 
cross/inner